pysilcam/postprocess.py

```diff
@@ -90,9 +90,9 @@ def get_size_bins():
     bin_mids_um[0] = 2.72
 
     # loop through 53 size classes and calculate the bin mid-points
-    for i in np.arange(1, 52, 1):
+    for bin_number in np.arange(1, 52, 1):
         # each bin is 1.18 * larger than the previous
-        bin_mids_um[i] = bin_mids_um[i - 1] * 1.180
+        bin_mids_um[bin_number] = bin_mids_um[bin_number - 1] * 1.180
```

Comment:
Perhaps instead: bin_mids_um[1:] = bin_mids_um[:-1] * 1.180

Reply:
not sure i understand. bin_number is an iterator in the for loop.